sjb/config/test_cases/test_branch_ansible_service_broker.yml

```diff
@@ -0,0 +1,48 @@
+---
+parent: 'test_cases/test_branch_origin_service_catalog.yml'
+overrides:
+  sync:
+    - "openshift,ansible-service-broker=master"
```

Comment:
I don't think you want any overrides of sync. The extensions part should be all you need.